<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>731</width>
     <height>511</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>611</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>河南科技学院——物联网181第十组智能家居</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>341</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>组员：张博  王浩博  薛庆贺  董世龙 刘家威</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connBtn">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>320</y>
       <width>281</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>点击进入</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>主页面</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>230</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>数据界面</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>320</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>智能家居界面</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mainBtn1">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>230</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>点击进入</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mainBtn2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>310</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>点击进入</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mainquit">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>430</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="messageedit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>130</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>421</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>温湿度、烟雾浓度数据查看</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dataedit1">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>150</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="dataedit2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>220</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="dataedit3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>290</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>温度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>220</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>湿度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>280</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>烟雾浓度</string>
     </property>
    </widget>
    <widget class="QPushButton" name="secquit">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>430</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>321</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>智能家具控制界面</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>210</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>智能小灯</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>215</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>智能音响</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LEDOPEN">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>打开</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LEDOFF">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>270</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BUZZEROPEN">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>270</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>音乐1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BUZZEROFF">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>270</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>音乐2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="lastquit">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>430</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
